sh/kssh

```diff
@@ -39,13 +39,24 @@ fi
 # So we will do a second review to know if we are inside a directory
 # that contains the server's templates, or if we are in any other directory.
 REPOSITORY_PROFILES=
-for path_to_test in `find ~/ -name srv_tmpl.yml`
+CURRENT_DIR=$(pwd)
+for (( ; ; ))
 do
-  TEMPVAR=$(dirname $path_to_test)
-  if stringContains $TEMPVAR $('pwd')
-  then
-    REPOSITORY_PROFILES=$TEMPVAR/services_information/servers/
+  # Let's check if the reference file is in the current directory
+  if [[ -f $CURRENT_DIR/srv_tmpl.yml ]]; then
+    # If so, then assign that directory in the propper variable
+    REPOSITORY_PROFILES=$CURRENT_DIR/services_information/servers/
```

Comment:
`.../services_information/servers/` is normally defined in the srv_tmpl.yml file directly. we cannot assume to be in that path everytime.